aptos-move/framework/aptos-stdlib/sources/debug.move

```diff
@@ -1,6 +1,271 @@
 /// Module providing debug functionality.
 module aptos_std::debug {
-    native public fun print<T>(x: &T);
+    use std::string::String;
 
-    native public fun print_stack_trace();
+    public fun print<T>(x: &T) {
+        native_print(format(x));
```

Comment:
Nice refactoring to have this call the string formatting!

Reply:
Is this backwards compatible?

Reply:
I mean it's debug and not prod right?

Reply:
But I don't see the `#[test_only]` clauses...

Reply:
Debug module is not deployed to any live networks

Reply:
native public fun print_stack_track => public fun might still be backward compatible since both visibility and function signature didn't change. Removing native might be fine

Reply:
I added back the native as backup otherwise the thing could fork / crash.